packages/connect-thegraph-token-manager/package.json

```diff
@@ -7,16 +7,23 @@
   "scripts": {
     "build": "yarn clean && yarn compile",
     "dev": "yarn build --watch",
+    "test": "is-ci \"test:coverage\" \"test:watch\"",
```

Comment:
Since we're using `is-cli` here, we may also want to mark it as a dev dependency